buildpipeline/pipeline.json

```diff
@@ -31,6 +31,21 @@
           }
         },
         {
+          "Name": "Core-Setup-Linux-BT",
+          "Parameters": {
+            "PB_DistroRid": "rhel.6.9-x64",
```

Comment:
Thanks for pointing that out @weshaggard , I've renamed the PB_DistroRid value as suggested.